boards/arm/qemu_cortex_a53/qemu_cortex_a53.yaml

```diff
@@ -0,0 +1,21 @@
+identifier: qemu_cortex_a53
+name: QEMU Emulation for Cortex-A53
+type: qemu
+simulation: qemu
+arch: arm
+toolchain:
+  - zephyr
+  - cross-compile
+ram: 128
+testing:
+  ignore_tags:
```

Comment:
Why do you need all these ignore_tags? Is the port incomplete?

Reply:
Some tests are WiP blocked on other commits not strictly related to this port, others are waiting for some common work to be done on Cortex-R and Cortex-A when we will merge the common code and others are simply not supported at the moment (nothing major and mostly not related to the arch port).

-`cmsis_rtos`: CMSIS is not supported yet on Cortex-A
-`console`: to enable this we need #20982 and some more commits I have ready on top
-`crypto`: not supported yet
-`drivers`: blocked again on #20982 
-`interrupt`: the test doesn't support the fake cascade IRQ controller that we are currently using. Needs some common rework for Cortex-R and Cortex-A.
-`jwt`: this can be enabled, it is passing now
-`net`: blocked again on #20982 & co
-`nfc`: blocked on #20982 & co
-`nmi`: this can be enabled
-`shell`: blocked again on #20982